docs/Spark/configuration/configuration.md

```diff
@@ -65,6 +101,12 @@ In the below image `Config.num_top_customers` is fetching the integer value defi
 
 ![Config Limit Example](img/config-pipeline-limit-eg-scala-python.png)
 
+#### Jinja Config
+
```

Comment:
We should also add jinja syntax for array and record fields as well. For example :: for accessing elements of an array the jinja syntax is `{{ config1.array_config[23] }}`. Similarly, for records it should be like `{{ record1.record2.field1 }}`

Reply:
Done.